src/sentry/sentry_apps/api/endpoints/installation_external_issues.py

```diff
@@ -44,13 +44,16 @@ def post(self, request: Request, installation) -> Response:
 
         serializer = PlatformExternalIssueSerializer(data=request.data)
         if serializer.is_valid():
-            external_issue = ExternalIssueCreator(
-                install=installation,
-                group=group,
-                web_url=data["webUrl"],
-                project=data["project"],
-                identifier=data["identifier"],
-            ).run()
+            try:
+                external_issue = ExternalIssueCreator(
+                    install=installation,
+                    group=group,
+                    web_url=data["webUrl"],
+                    project=data["project"],
+                    identifier=data["identifier"],
+                ).run()
+            except Exception:
+                return Response({"error": "Failed to create external issue"}, status=400)
```

Comment:
I wonder if the best status here is 400 🤔 what are some reasons this might fail?

Reply:
🤔  I seees, yeah we do catch the client side errors in the serializer validation step. In the creation step because we create or update now, we wouldn't run into any integrity or not found errors. I think the most likely case we fail in this block is some kind of database error, so we'd probably want to return a 500.